src/hackerspace_online/celery.py

```diff
@@ -29,10 +29,14 @@
 # always attempt to run a schedule that is listed in the database even if
 # it is not listed here or the name of the task is changed.
 app.conf.beat_schedule = {
+    "Send daily email of noticiations to all schemas": {
+        "task": "notifications.tasks.email_notification_to_users_on_all_schemas",
+        "schedule": crontab(minute=0, hour=5),
+    },
     "Invalidate Profile Cache for all schemas daily task": {
         "task": "profile_manager.tasks.invalidate_profile_cache_in_all_schemas",
```

Comment:
I know this was a previous PR, but this needs a better name.  We do not cache profiles, so we are not invalidating profile caches.  What does this method actually do? it should be named to match what it's doing.